include/maliput/utility/generate_dot.h

```diff
@@ -45,5 +46,18 @@ namespace utility {
 /// @throws common::assertion_error When @p os is nullptr.
 void GenerateDotStream(const routing::graph::Graph& graph, std::ostream* os);
 
+/// Generates a string representation in @p os of the @p graph and @p route using the DOT language.
```

Comment:
I see on line 56 that the route is highlighted. Can you add a sentence after this one that emphasizes that the route is highlighted and describe how it is highlighted? For example, is it highlighted in boldface or in a different color?

(After looking at the .cc file, I see that the route is highlighted in red, but I would like to know that without looking at the .cc file.)